pkg/convert/testdata/programs/expressions/pcl/main.pp

```diff
@@ -88,7 +88,15 @@
 }
 
 output "heredoc" {
-  value = "This is also a template.\nSo we can output the key again ${aKey}\n"
+  value = <<END
+This is also a template.\nSo we can output the key again ${aKey}\n
```

Comment:
Won't this keep \n in the final result now because it's now wrapped in a HEREDOC?

Reply:
great catch, didn't even notice.  Fixed now